src/components/Search/index.js

```diff
@@ -12,13 +13,13 @@ class Search extends React.Component {
   handleSubmit = (e) => {
     e.preventDefault();
     this.props.history.push(`/search/${this.state.value}`);
+    this.setState({ value: "" });
   };
   render() {
     return (
       <>
         <form onSubmit={this.handleSubmit}>
```

Comment:
Do you need to nest the form element into a nameless component? I guess you could use form as the main component right?

Reply:
Good point!